postal/service.go

```diff
@@ -129,14 +130,15 @@ func (s Service) Resolve(path, id, version, stack string) (Dependency, error) {
 	return compatibleVersions[0], nil
 }
 
-// Install will fetch and expand a dependency into a layer path location. The
+// Deliver will fetch and expand a dependency into a layer path location. The
 // location of the CNBPath is given so that dependencies that may be included
-// in a buildpack when packaged for offline consumption can be retrieved. The
-// dependency is validated against the checksum value provided on the
-// Dependency and will error if there are inconsistencies in the fetched
-// result.
-func (s Service) Install(dependency Dependency, cnbPath, layerPath string) error {
-	dependencyMappingURI, err := s.mappingResolver.FindDependencyMapping(dependency.SHA256, "/platform/bindings")
+// in a buildpack when packaged for offline consumption can be retrieved. If
+// there is a dependency mapping for the specified dependency, Deliver will use
+// the given dependency mapping URI to fetch the dependency. The dependency is
+// validated against the checksum value provided on the Dependency and will
+// error if there are inconsistencies in the fetched result.
+func (s Service) Deliver(dependency Dependency, cnbPath, layerPath, platformPath string) error {
+	dependencyMappingURI, err := s.mappingResolver.FindDependencyMapping(dependency.SHA256, filepath.Join(platformPath, "bindings"))
```

Comment:
I didn't realize that the platform path could be anything other than `/platform`, but I need see in the spec that that's just a default value. Random question - at what point could the platform path be set?

Reply:
Arguably a platform that isn't `pack` or `kpack` could set a different filepath as the Platform path when it invokes the lifecycle and then hardcoding this like we have wouldn't work.